gdal/ogr/ogrsf_frmts/sqlite/ogrsqlitevirtualogr.cpp

```diff
@@ -1713,6 +1713,7 @@ static const struct sqlite3_module sOGR2SQLITEModule =
     nullptr,  // xSavepoint
     nullptr,  // xRelease
     nullptr,  // xRollbackTo
+    nullptr,  // xShadowName
```

Comment:
unrelated change. breaks some build like the ubuntu_1804 of https://travis-ci.org/OSGeo/gdal/builds/476518997